adapters/bidmachine/bidmachine_test.go

```diff
@@ -0,0 +1,20 @@
+package bidmachine
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderBidmachine, config.Adapter{
+		Endpoint: "http://{{.Host}}.example.com/prebid"})
+
+	if buildErr != nil {
+		t.Fatalf("Builder returned unexpected error %v", buildErr)
+	}
+
+	adapterstest.RunJSONBidderTest(t, "bidmachinetest", bidder)
+}
```

Comment:
Please add the `TestEndpointTemplateMalformed` test from https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html#builder-tests.